htdocs/fourn/class/fournisseur.commande.class.php

```diff
@@ -837,6 +838,9 @@ public function getNomUrl($withpicto = 0, $option = '', $notooltip = 0, $save_la
 
 		$label = '';
 
+		$fourn = new Societe($this->db);
```

Comment:
A getNomUrl method should not contains any subrequests. The method is often used to output long list and introducing a subrequest into a formating method will lead to very bad performances.
You can add the supplier, but the information must already be saved inside the $object (so into $object->thirdparty->name must be set by caller)